src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int length;
+    private int width;
+
+    public Rectangle(int length, int width, String color) {

```

Comment:
Consider using the `Color` enum instead of a `String` for the color parameter to maintain type safety and consistency with the rest of the code.